wix/main.wxs

```diff
@@ -44,12 +42,11 @@
             Keywords='Installer'
             Description='The JavaScript Launcher'
             Manufacturer='The Volta Maintainers'
-            InstallerVersion='450'
+            InstallerVersion='500'
             Languages='1033'
             Compressed='yes'
             InstallScope='perMachine'
-            SummaryCodepage='1252'
-            Platform='$(var.Platform)'/>
```

Comment:
Why are you dropping the `Platform` value here?

Reply:
I am deleting it for the same reason as here: https://github.com/volta-cli/volta/pull/1801#discussion_r1687443667